### PR TITLE
[bitnami/kafka] adding extra rules to jmx template

### PR DESCRIPTION
**Pull request by davidballano**
### Description of the change

Adding a config called extrarules that defauls to "" that will add extra jmx rules to jmx-configmap.yaml template

### Benefits

User will be able to add extra metrics to monitor Kafka from their values.yaml without having to either create a new configmap or modify the chart template jmx-configmap.yaml

### Possible drawbacks
There should not be any drawbacks, if the setting is not used the default rules will still apply.
### Applicable issues
NA

### Additional information
I couldnt get readme-generator-for-helm to work to generate the README

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by fmulero — CHANGES_REQUESTED** [review]
Hi @davidballano. Thanks a lot for your contribution, it looks great!

I left some comments, Could you take them a look?

File: bitnami/kafka/values.yaml [hunk 1]
```diff
@@ -1391,6 +1391,9 @@ metrics:
     ## NOTE: This will override metrics.jmx.config
     ##
     existingConfigmap: ""
+    ## @param metrics.jmx.extrarules  Add extra rules to JMX exporter configuration
+    ##
+    extrarules: ""
```
[the remark this answers is not in the document]
Comment (davidballano) [reply bitnami/kafka/values.yaml]: sure thing.

File: bitnami/kafka/Chart.yaml [hunk 2]
```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/kafka
   - https://kafka.apache.org/
-version: 18.1.1
+version: 18.2.1
```
[the remark this answers is not in the document]
Comment (davidballano) [reply bitnami/kafka/Chart.yaml]: my mistake.

**Review by fmulero — APPROVED** [review]
Great!, LGTM. Thanks a lot @davidballano

**Review by fmulero — APPROVED** [review]

**Pull request merged**